functional_tests/steps/page_editor.py

```diff
@@ -127,3 +127,13 @@ def the_user_hides_the_minimap(context: Context):
 @then("the minimap is hidden")
 def the_minimap_is_hidden(context: Context):
     expect(context.page.get_by_role("complementary", name="Minimap").locator("div").first).not_to_be_visible()
+
+
+@when('the user clicks "Save" to save the Snippet')
+def user_saves_in_navigation_settings(context: Context):
+    context.page.get_by_role("button", name="Save").click()
+
+
+@when("the user clicks toggle preview ")
```

Comment:
Nit-pick: Dangling space
```suggestion
@when("the user clicks toggle preview")
```

Reply:
Done [7f23b29](https://github.com/ONSdigital/dis-wagtail/pull/130/commits/7f23b29a24646eef450fe1de8e0d50b906b29a50)